assimilation_code/modules/assimilation/assim_tools_mod.f90

```diff
@@ -1084,21 +1060,6 @@ subroutine obs_increment(ens_in, ens_size, obs, obs_var, obs_kind, obs_inc, &
 ! Add in the extra increments if doing observation space covariance inflation
```

Comment:
Comment for line 1008

```
   if(filter_kind == 1) then
      call obs_increment_eakf(ens, ens_size, prior_mean, prior_var, &
         obs, obs_var, obs_inc, net_a)
   else if(filter_kind == 2) then
      call obs_increment_enkf(ens, ens_size, prior_var, obs, obs_var, obs_inc)
   else if(filter_kind == 3) then
      call obs_increment_kernel(ens, ens_size, obs, obs_var, obs_inc)
   else if(filter_kind == 4) then
      call obs_increment_particle(ens, ens_size, obs, obs_var, obs_inc)
   else if(filter_kind == 8) then
      call obs_increment_rank_histogram(ens, ens_size, prior_var, obs, obs_var, obs_inc)
   else if(filter_kind == 11) then
```

use the public parameters from algorithm_info_mod.f90
https://github.com/NCAR/DART/blob/5c03e3cdd5332510a0532b02fcbb15d041a8cc88/assimilation_code/modules/assimilation/algorithm_info_mod.f90#L34-L38